agent/core/src/test/java/org/jolokia/handler/ListHandlerTest.java

```diff
@@ -168,7 +168,7 @@ public void truncatedList() throws Exception {
         JmxListRequest request = new JmxRequestBuilder(RequestType.LIST).pathParts("java.lang", "type=Runtime").build();
         Map res = execute(request);
         assertFalse(res.containsKey("op"));
-        assertEquals(res.size(),2);
+        assertEquals(res.size(),3);
```

Comment:
Should be fine. The only key missing here is "op" as the RuntimeMBean does not have an operations.

Would be cool if you could update the docs, too (and maybe replace "Reduce the depth of depth" by "Reduce the depth of the tree" while you are at it ;-)